### PR TITLE
Update PullApprove configuration

### PR DESCRIPTION
**Pull request by aserv92**
Rename the PullApprove configuration. Modify PullApprove conditions and add unmet status explanations.

### CODE REVIEW

**Comment by aserv92** [comment]
!force code review

**Comment by healthlabs-bugman** [comment]
Testing file changes were not found.
Paging @ramsey-medology to review the back-end work in this pull request!
Paging @smijo149 to review the testing work in this pull request!

I couldn't recognize who these files were relevant to:
.pullapprove.yml

**Comment by aserv92** [comment]
Forced code review because I broke bugman just before creating this PR.

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 3
+extends: "https://api.github.com/repos/medology/Scripts/contents/Pull%20Approve/base.pullapprove.yml?ref=master"
+pullapprove_conditions:
+  - 
+    condition: "base.ref == 'master'"
+    unmet_status: failure
+    explanation: "PullApprove is not configured for this branch"
```
Comment (smijo149) [inline .pullapprove.yml]: As per the provided [documentation](https://docs.pullapprove.com/config/pullapprove-conditions/), `-` doesn't have to be in a separate line. So I would suggest to follow their doc and write it as,
```
 - condition: "base.ref == 'master'"
   unmet_status: failure
   explanation: "PullApprove is not configured for this branch"
```

File: .pullapprove.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - 
+    condition: "'[DO NOT REVIEW]' not in title"
+    unmet_status: pending
+    explanation: "Not ready for review"
+  - 
+    condition: "state == 'open'"
+    explanation: "PR must be open to be reviewed"
+    unmet_status: failure
```
Comment (smijo149) [inline .pullapprove.yml]: I would suggest to follow the same format throughout the file.
```yml
condition: ...
unmet_status: ...
explanation: ...
```

File: .pullapprove.yml (hunk 1 above)
Comment (aserv92) [reply .pullapprove.yml]: Moved to the same line as condition

File: .pullapprove.yml (hunk 2 above)
Comment (smijo149) [reply .pullapprove.yml]: Here I meant the order of `condition`, `unmet_status` and `explanation`.  Sorry for the confusion.

**Comment by aserv92** [comment]
Abandoning this PR as the master branch of this repo is abandoned.

**Pull request closed without merging**